build/rpms/10-kubeadm.conf

```diff
@@ -9,5 +9,4 @@ Environment="KUBELET_AUTHZ_ARGS=--authorization-mode=Webhook --client-ca-file=/e
 Environment="KUBELET_CGROUP_ARGS=--cgroup-driver=systemd"
 Environment="KUBELET_CADVISOR_ARGS=--cadvisor-port=0"
 Environment="KUBELET_CERTIFICATE_ARGS=--rotate-certificates=true"
-ExecStart=
```

Comment:
why ? another question, do you have corresponding PR to sync such change to release repository ? this drop-in file should be in sync in two places for now.

Reply:
1.unit file ExecStart = did not execute anything
2.I sync this change to release repository

Reply:
https://github.com/kubernetes/release/pull/511

Reply:
`ExecStart=` line is present to re-set list of commands to execute. So, if you remove that line, it means it will try to execute kubelet twice, which is not what we want. 
```
kad:/home/kad # systemctl show kubelet | grep ExecStart=
ExecStart={ path=/usr/bin/kubelet ; argv[]=/usr/bin/kubelet ; ignore_errors=no ; start_time=[n/a] ; stop_time=[n/a] ; pid=0 ; code=(null) ; status=0/0 }
ExecStart={ path=/usr/bin/kubelet ; argv[]=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_SYSTEM_PODS_ARGS $KUBELET_NETWORK_ARGS $KUBELET_DNS_ARGS $KUBELET_AUTHZ_ARGS $KUBELET_CADVISOR_ARGS $KUBELET_CGROUP_ARGS $KUBELET_CERTIFICATE_ARGS $KUBELET_EXTRA_ARGS ; ignore_errors=no ; start_time=[n/a] ; stop_time=[n/a] ; pid=0 ; code=(null) ; status=0/0 }
kad:/home/kad #
```

So, you should not remove it.